packages/common/src/models/gradtrak.ts

```diff
@@ -0,0 +1,202 @@
+import mongoose, { Document, InferSchemaType, Schema } from "mongoose";
+
+export const GradtrakCustomEventSchema = new Schema({
+  start_time: {
+    type: String,
+    required: true,
+    trim: true,
+    alias: "start",
+  },
+  end_time: {
+    type: String,
+    required: true,
+    trim: true,
+    alias: "end",
+  },
+  title: {
+    type: String,
+    required: false,
+    alias: "name",
+  },
+  location: {
+    type: String,
+    required: false,
+    trim: true,
+    alias: "place",
+  },
+  description: {
+    type: String,
+    required: false,
+  },
+  days_of_week: {
+    type: String,
+    required: false,
+    trim: true,
+    alias: "days",
+  },
+  uni_reqs: {
+    type: [String],
+    required: false,
+    trim: true,
+  },
+  college_reqs: {
+    type: [String],
+    required: false,
+    trim: true,
+  }
+});
+
+export const selectedCourseSchema = new Schema({
+  class_ID: {
```

Comment:
If you are storing classes, then you should probably reference the schedule schema. If you are only storing courses (which is probably the only thing necessary to start), then you only need to store the `courseId` and not any of the class or section identifiers.